AWSSDKSwiftCLI/Sources/AWSCLIUtils/URL+Path.swift

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+@_disfavoredOverload
+public func urlPath(_ url: URL?) -> String? {
+    guard let url else { return nil }
+    return urlPath(url)
+}
```

Comment:
It seems that on our supported Mac versions of Swift, you get the path from a URL with the `path(percentEncoded:)` method, while on Linux Foundation they still only have the `.path` property.

These two helper methods use the right one for the platform in use.